uktrade/data-hub-leeloo
# Rename large capital investor profile db table

### Description of change
Initially the database model for investor profiles was designed in a way that both large and growth capital profiles could be stored in. But this was considered not necessary and after more user research it was determined they would vary too much.

This PR renames the django model and database tables used to store the Large Capital Investor Profiles. Factories/Tests have all been updated to be more consistent. It is worth noting that this feature is not yet live so there is no data in the old database table.

### To test
- Ensure you have the latest Frontend code running
- Go to http://localhost:3000/companies/375094ac-f79a-43e5-9c88-059a7caa17f0/investments/large-capital-profile
- Create a profile


### Checklist
